lib/bump/update_checkers/python.rb

```diff
@@ -1,15 +1,20 @@
 # frozen_string_literal: true
 require "excon"
 require "bump/update_checkers/base"
+require "bump/shared_helpers"
 
 module Bump
   module UpdateCheckers
     class Python < Base
       def latest_version
         @latest_version ||=
           begin
-            url = "https://pypi.python.org/pypi/#{dependency.name}/json"
-            JSON.parse(Excon.get(url).body)["info"]["version"]
+            pypi_response = Excon.get(
+              dependency_url,
+              middlewares: SharedHelpers.excon_middleware
+            )
+
+            JSON.parse(pypi_response.body)["info"]["version"]
```

Comment:
Yeah, I was surprised the existing code didn't handle that. Have you not been receiving sentries about it?